src/stores/searchContext.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { createContext, useContext, useState } from 'react'
+
+export type SearchContextType = {
+  searchQuery: string
+  setSearchQuery: (searchQuery: string) => void
+}
+
+export const SearchContext = createContext<SearchContextType>({
+  searchQuery: '',
+  setSearchQuery: /* istanbul ignore next */ () => {
+    return
+  },
+})
+
+export const SearchProvider = ({ children }: { children: React.ReactNode }) => {
+  const [searchQuery, setSearchQuery] = useState('')
+
+  const context = {
+    searchQuery,
+    setSearchQuery,
+  }
+
+  return (
+    <SearchContext.Provider value={context}>{children}</SearchContext.Provider>
+  )
+}
```

Comment:
I thought it would be beneficial to create a provider for search. First, I needed it so that the `SearchFilter` component could be aware of the current value that is in the `Search` component. But thinking about it a little more, I think it will beneficial in the future as well when we add a filter option to the `/news` page. Let me know if there is a better approach! I know we already have a handful of providers, so definitely don't want to be creating them unnecessarily.